shared/src/packages/users/libs/types/user-sign-up-response-dto.type.ts

```diff
@@ -1,6 +1,7 @@
 type UserSignUpResponseDto = {
   id: number;
   email: string;
+  token: string;
```

Comment:
Let's send token separately. Something like this:
{
    user: UserSignUpResponseDto;
    token: string
}

Some other guys already implemented it like this, please, check their PRs

Reply:
Got it, will try to find it